dist/types/index.d.ts

```diff
@@ -203,8 +203,8 @@ export declare class IssuerHandler extends WorkerEntrypoint<Bindings> {
 		status?: number;
 		responseContentType: string;
 	}>;
-	rotateKey(url: string, prefix: string): Promise<Response>;
-	clearKey(url: string, prefix: string): Promise<Response>;
+	rotateKey(url: string, prefix: string): Promise<Uint8Array>;
+	clearKey(url: string, prefix: string): Promise<string[]>;
```

Comment:
should this use Uint8Array[] instead for consistency?

Reply:
These are [R2Object](https://developers.cloudflare.com/r2/api/workers/workers-api-reference/#r2object-definition) keys. So they are already strings

Reply:
true. there is a prefix now. this used to be the public key